page-factory-core/src/main/java/ru/sbtqa/tag/pagefactory/capabilities/SelenoidCapabilitiesParser.java

```diff
@@ -27,16 +27,14 @@ public DesiredCapabilities parse() {
         setCapability("enableLog", PROPERTIES.getSelenoidEnableLog());
         setCapability("screenResolution", PROPERTIES.getSelenoidScreenResolution());
         setCapability("enableVideo", PROPERTIES.getSelenoidEnableVideo());
-        setCapability("videoName", PROPERTIES.getSelenoidVideoName(), VIDEONAME_FORMAT);
         setCapability("videoScreenSize", PROPERTIES.getSelenoidVideoScreenSize());
         setCapability("videoFrameRate", PROPERTIES.getSelenoidVideoFrameRate());
-        String selenoidName;
-        if (PROPERTIES.getSelenoidNameOfTests().trim().isEmpty() && Environment.getScenario() != null) {
-            selenoidName = Environment.getScenario().getName();
-        } else {
-            selenoidName = PROPERTIES.getSelenoidNameOfTests();
-        }
-        setCapability("name", selenoidName);
+        String name = getProperty(PROPERTIES.getSelenoidNameOfTests());
```

Comment:
Declare a final strings in a separate block on top of this method.
Also, move all concatenations to declaration block and use pure variables only in set method.